src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -155,6 +155,17 @@ protected static IOutput Output
         /// </summary>
         public Verbosity VerbosityLevel => verbosityLevel;
 
+        public class Summary
+        {
+            public int Totaltests { get; set; }
+            public int Passedtests { get; set; }
+            public int Failedtests { get; set; }
+            public int Skippedtests { get; set; }
+            public TimeSpan timespan { get; set; }
```

Comment:
Why is this lower cased?
Any reason why all these are properties ?

Reply:
resolved